.changeset/smooth-cycles-hear.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/theme-check-common': minor
+'@shopify/theme-check-node': minor
```

Comment:
I know we technically didn't modify it, but I like to see those added to the vscode extension readme when they are added. 

Mind adding `theme-check-vscode: minor` in there?